src/components/JsonSchema/index.tsx

```diff
@@ -138,6 +138,11 @@ export const JsonSchemaForm = ({
             onChange={onChange}
             formError={formError}
             setFormError={setFormError}
+            parsedSorobanOperation={
```

Comment:
we pass parsed soroban related operation from the parent so that we could re-use this in smart contract page's invoke contract